_positions/product-manager.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Product Manager
+permalink: positions/product-manager/
+team: 18F
+active: true
+---
+##Basic Information:
+**Title:** Product Manager  <br>
+**Series & Grade Level:** GS-2210-14/15 <br>
+**Salary Range:** Base Salary GS-14 - $87,263 to $113,444; Base Salary GS-15 - $102,646 to $133,444. Total compensation will include locality pay based on the individual's duty location. For locality pay details, please visit: https://www.opm.gov/policy-data-oversight/pay-leave/salaries-wages/ <br>
+**Work Schedule:** Full Time<br>
+
+**Job Announcement Number:** 17001-Product Manager<br>
+**Open:** DATE<br>
+**Close:** DATE<br>
+
+**Who May Apply:**
+Any individual who is not currently a GSA employee or contractor and who is a U.S. Citizen, a non-citizen national of the U.S., or has been admitted to the U.S. for permanent residence and holds a valid green card is eligible to apply.
+
+**Position Type:** Excepted Service - Not to exceed 24 months (2 years); may be extended for an additional 2 years for a maximum of 4 years total.<br>
+**Number of Vacancies:** 3 (At this time, we are only filling one vacancy however additional vacancies may be filled from this announcement as needed)<br>
+**Location:** Anywhere in the United States<br>
+**Travel Required:** Occasional Travel<br>
+**Supervisory Status:** No<br>
+**Security Clearance:** Public Trust - background investigation required.<br>
+
+Do you have a passion for public service? The General Services Administration (GSA) is looking for you! GSA is a federal government agency whose mission is to bring the best value in real estate, acquisition, and technology services to the American people.<br>
```

Comment:
Sure, we can add something at the end like "18F is an organization inside GSA's branch of technology (Technology Transformation Service (TTS))".